src/Arcus.Template.WebApi/Startup.cs

```diff
@@ -22,6 +26,8 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
             #warning "Please configure application with authentication mechanism: https://webapi.arcus-azure.net/features/security/auth/shared-access-key"
 
             app.UseMvc();
+
+            app.UseHealthChecks("/health");
```

Comment:
We don't really need this as we'll expose the endpoint ourselves based on the health middleware. See https://www.codit.eu/blog/documenting-asp-net-core-health-checks-with-openapi/ for more info